packages/guides-restructured-text/src/RestructuredText/Parser/Productions/ListRule.php

```diff
@@ -59,7 +58,6 @@ final class ListRule implements Rule
 
     public function __construct(
         private readonly RuleContainer $productions,
-        private readonly LoggerInterface $logger,
```

Comment:
I thinks we should not remove this right now, it's not part of the change, and I see removing an argument as a breaking change. Also because a change in the tests is required. If this leads to an unused property, we could transform it to a normal parameter, and make it optional. 
Add a deprecation and remove it in v2.

Reply:
I just added it to the phpstan baseline. we might need it again , there is no harm in leaving it in